backtracking/sudoku_solve.cpp

```diff
@@ -1,9 +1,6 @@
 #include <iostream>
-using namespace std;
-/// N=9;
-int n = 9;
 
-bool isPossible(int mat[][9], int i, int j, int no) {
+bool isPossible(int mat[][9], int i, int j, int no, int n) {

```

Comment:
this C-style array might raise an error.

Reply:
Shall I use an array or vector?

Reply:
pointers or `std::array` would be preferable. 
But the main thing is to maintain consistency otherwise the code execution would run into memory access issues.